apps/web/languages/en.json

```diff
@@ -18,6 +18,11 @@
         "title": "Have your own <0>person space</0>, and use it however your want.",
         "subtitle": "Create an account to save and store everything you do on the site.",
         "aboutTheSiteButton": "About the site"
+      },
+      "utility": {
+        "signInButton": "Sign in",
+        "signUpButton": "Sign up",
+        "popoverTitle": "Settings"
```

Comment:
@MFelipeSilva I think is better to create another object inside utility object, ex. `"popover": { "title": "Settings" }`, what u think?

Reply:
corrected